openapi/yarn.lock

```diff
@@ -2,24 +2,86 @@
 # yarn lockfile v1
 
 
-"@okta/openapi@^2.11.0":
-  version "2.11.1"
-  resolved "https://registry.yarnpkg.com/@okta/openapi/-/openapi-2.11.1.tgz#49e9738fbd68520b7a2deaf74d63f67e41a3e924"
-  integrity sha512-FdDTwscwULg94nGFAhlncLpKr+46nfkjWe2npe9+Th8xaMcnRYtymsUMI8wXNMrOxOJ9+lxTplnfJVl1n4i/Cw==
+"@apidevtools/json-schema-ref-parser@9.0.6":
```

Comment:
Left `yarn.lock` changes in as it seemed the `yarn build` ran consistently faster with them